### ISSUE TITLE
Conflagrate-Chalice Of The Void Bug 

### ISSUE THREAD

**Issue by mrsqrrl** [magefree/mage#2436]
Hey, I've repeatedly gotten my flashed back Conflagrates for x = >0 countered by a Chalice of the Void with one charge counter on it. As far as I know the cmc of a flashed back Conflagrate is 1 plus twice the number of cards discarded. Is there something that can be done?

**Comment by spjspj** [comment]
X=what you chose , so CMC on the stack for flashbacked conflagrate should be 2X + 1 so probably a bug there :(

```
117.9c An alternative cost doesn't change a spell's mana cost, only what its controller has to pay to cast it. Spells and abilities that ask for that spell's mana cost still see the original value.
```

**Comment by Navyguy330** [comment]
the cmc for that card can only be odd, it also is X so no matter how many cards u discard (as long as its more than 0) it will alwyas be that number plus 1, so a chalice for one cannot counter that unless u choose zero both time (casting and flashback)

**Comment by Navyguy330** [comment]
a chalice for 2 can never counter it either, the min cmc for that card is 1 or 3 and above in increments of odd numbers

**Comment by ImperatorPrime** [comment]
Fixed by 900d68f77defab5260fb27ee442689abd08391f6
